scripts/src/public/services/orders.ts

```diff
@@ -78,8 +78,9 @@ export async function getOrder(orderId: string, logger: LoggerInstance): Promise
 	return orderDbToApi(order);
 }
 
-export async function getOpenOrder(orderId: string): Promise<db.MarketplaceOrder| db.ExternalOrder> {
-	const order = await db.Order.getOne(orderId, "opened") as db.MarketplaceOrder | db.ExternalOrder;
+export async function getOrderForWhitelisting(orderId: string): Promise<db.MarketplaceOrder| db.ExternalOrder> {
+	// Get order, regardless of status
```

Comment:
why regardless of status? didn't we agree on open or pending?

Reply:
Need to write a new method that accepts a list of statuses, and it really doesn't matter what the status of the order is